This is a testharness.js-based test.
PASS dedicatedworker-connect-src
PASS Same-origin 'fetch()' in http: with connect-src 'self'
PASS Same-origin XHR in http: with connect-src 'self'
PASS Cross-origin 'fetch()' in http: with connect-src 'self'
PASS Cross-origin XHR in http: with connect-src 'self'
PASS Same-origin => cross-origin 'fetch()' in http: with connect-src 'self'
FAIL Reports match in http: with connect-src 'self' assert_array_equals: Reports do not match lengths differ, expected array ["http://web-platform.test:8001/common/redirect-opt-in.py?status=307&location=https://www.web-platform.test:8445/content-security-policy/support/resource.py?cross-origin-fetch", "https://www.web-platform.test:8445/content-security-policy/support/resource.py?cross-origin-fetch", "https://www.web-platform.test:8445/content-security-policy/support/resource.py?cross-origin-xhr"] length 3, got [] length 0
PASS Same-origin 'fetch()' in blob: with connect-src 'self'
PASS Same-origin XHR in blob: with connect-src 'self'
PASS Cross-origin 'fetch()' in blob: with connect-src 'self'
PASS Cross-origin XHR in blob: with connect-src 'self'
PASS Same-origin => cross-origin 'fetch()' in blob: with connect-src 'self'
FAIL Reports match in blob: with connect-src 'self' assert_array_equals: Reports do not match lengths differ, expected array ["http://web-platform.test:8001/common/redirect-opt-in.py?status=307&location=https://www.web-platform.test:8445/content-security-policy/support/resource.py?cross-origin-fetch", "https://www.web-platform.test:8445/content-security-policy/support/resource.py?cross-origin-fetch", "https://www.web-platform.test:8445/content-security-policy/support/resource.py?cross-origin-xhr"] length 3, got [] length 0
PASS Same-origin 'fetch()' in filesystem: with connect-src 'self'
PASS Same-origin XHR in filesystem: with connect-src 'self'
PASS Cross-origin 'fetch()' in filesystem: with connect-src 'self'
PASS Cross-origin XHR in filesystem: with connect-src 'self'
PASS Same-origin => cross-origin 'fetch()' in filesystem: with connect-src 'self'
FAIL Reports match in filesystem: with connect-src 'self' assert_array_equals: Reports do not match lengths differ, expected array ["http://web-platform.test:8001/common/redirect-opt-in.py?status=307&location=https://www.web-platform.test:8445/content-security-policy/support/resource.py?cross-origin-fetch", "https://www.web-platform.test:8445/content-security-policy/support/resource.py?cross-origin-fetch", "https://www.web-platform.test:8445/content-security-policy/support/resource.py?cross-origin-xhr"] length 3, got [] length 0
PASS Same-origin 'fetch()' in http: without CSP
PASS Same-origin XHR in http: without CSP
FAIL Cross-origin 'fetch()' in http: without CSP promise_test: Unhandled rejection with value: object "TypeError: Failed to fetch"
FAIL Cross-origin XHR in http: without CSP promise_test: Unhandled rejection with value: "xhr.open should success."
FAIL Same-origin => cross-origin 'fetch()' in http: without CSP promise_test: Unhandled rejection with value: object "TypeError: Failed to fetch"
Harness: the test ran to completion.

